src/opensynth/models/faraday/gaussian_mixture/fit_gmm.py

```diff
@@ -0,0 +1,160 @@
+# Copyright Contributors to the Opensynth-energy Project.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This script fits a Gaussian Mixture Model (GMM) to synthetic data using
+Pytorch Lightning. Code is based on the PyCave framework.
+"""
+
+import time
+from typing import Tuple
+
+import numpy
+import pytorch_lightning as pl
+from pytorch_lightning.loggers import CSVLogger
+from torch.utils.data import DataLoader
+
+from opensynth.models.faraday.gaussian_mixture.fit_kmeans import fit_kmeans
+from opensynth.models.faraday.gaussian_mixture.gmm_lightning import (
+    GaussianMixtureInitLightningModule,
+    GaussianMixtureLightningModule,
+)
+from opensynth.models.faraday.gaussian_mixture.model import (
+    GaussianMixtureModel,
+)
+
+logger = CSVLogger("lightning_logs", name="gmm_logs")
```

Comment:
I changed the logger to also record logs inside the `lightning_logs` directory. This means we save all the logs from the vae and kmeans and gmm in separate folders inside `lightning_logs`

Reply:
Perfect, ty!